node/node-tests.ts

```diff
@@ -423,6 +423,8 @@ function stream_readable_pipe_test() {
     var r = fs.createReadStream('file.txt');
     var z = zlib.createGzip({ finishFlush: zlib.Z_FINISH });
     var w = fs.createWriteStream('file.txt.gz');
+    console.log(r.bytesRead);
+    console.log(r.path);
```

Comment:
You're right, I wasn't sure what these test should actually achieve.

I'll add tests to check the actually types during runtime